src/scalar.rs

```diff
@@ -961,6 +961,80 @@ impl Scalar {
         output
     }
 
+    /// Creates a representation of a Scalar in radix 64, 128 or 256 for use with the Pippenger algorithm.
+    /// For lower radix, use `to_radix_16`, which is used by the Straus multi-scalar multiplication.
+    /// Higher radixes are not supported to save cache space. Radix 256 is near-optimal even for very
+    /// large inputs.
+    ///
+    /// Radix below 64 or above 256 is prohibited.
+    /// This method returns digits in a fixed-sized array, excess digits are zeroes.
+    /// The second returned value is the number of digits.
+    ///
+    /// ## Scalar representation
+    ///
+    /// Radix \\(2\^w\\), with \\(n = ceil(256/w)\\) coefficients in \\([-(2\^w)/2,(2\^w)/2)\\),
+    /// i.e., scalar is represented using digits \\(a\_i\\) such that
+    /// $$
+    ///    a = a\_0 + a\_1 2\^1w + \cdots + a_{n-1} 2\^{w*(n-1)},
+    /// $$
+    /// with \\(-2\^w/2 \leq a_i < 2\^w/2\\) for \\(0 \leq i < (n-1)\\) and \\(-2\^w/2 \leq a_{n-1} \leq 2\^w/2\\).
+    ///
+    pub(crate) fn to_pippenger_radix(&self, w: usize) -> ([i8; 43], usize) {
```

Comment:
```suggestion
    pub(crate) fn to_radix_2w(&self, w: usize) -> ([i8; 43], usize) {
```
would match the naming scheme for `to_radix_16` better?

Reply:
`2w` also reads as `2*w`. I intentionally placed "pippenger" there to bring attention to it being highly specialized and not accepting arbitrary `w` as an input. Any moment someone needs a slightly more general-purpose implementation, they should create a new method and not use this one.

How about `to_radix_64_128_256`?

Reply:
I tried a few variants, and `to_radix_2w` is the cleaner. It's private API anyway.